### PR TITLE
wtfutil: init at 0.19.1

### PR DESCRIPTION
**Pull request by yrashk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/wtfutil/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  modBuildPhase = ''
+    runHook preBuild
+    make build -mod=vendor
+    runHook postBuild
+  '';
+
+  meta = with stdenv.lib; {
+    description = "The personal information dashboard for your terminal.";
+    homepage = https://wtfutil.com;
+    license = licenses.mpl2;
```
Comment (jonringer) [inline pkgs/tools/misc/wtfutil/default.nix]:
```suggestion
    license = licenses.mpl20;
```

Comment (yrashk) [reply pkgs/tools/misc/wtfutil/default.nix]: Sure!

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
Diff LGTM (But also not super familiar with Go packaging)
binary works
leaf package

```
https://github.com/NixOS/nixpkgs/pull/66695
1 package were build:
wtfutil
```

**Review by marsam — COMMENTED** [review]
It's already packaged as [`wtf`](https://github.com/NixOS/nixpkgs/blob/31a577c7362a6ac40e40897dd623c7ad829a81b4/pkgs/applications/misc/wtf/default.nix)

**Comment by yrashk** [comment]
Moved to #66700

**Pull request closed without merging**